ReleaseTooling/Sources/FirebaseReleaser/Push.swift

```diff
@@ -80,7 +88,29 @@ enum Push {
       """)
     case let .success(output):
       print(output)
-      return output.trimmingCharacters(in: .whitespacesAndNewlines)
+      let repoName = output.trimmingCharacters(in: .whitespacesAndNewlines)
+      return repoName.isEmpty ? nil : repoName
+    }
+  }
+
+  /// @param defaultRepoName The repo name to register if not exists
+  private static func findOrRegisterPrivateCocoaPodsRepo(repo: String, gitRoot: URL,
+                                                         defaultRepoName: String) -> String {
+    if let repoName = findPrivateCocoaPodsRepo(repo: repo, gitRoot: gitRoot) {
+      return repoName
+    }
+
+    let command = "pod repo add \(defaultRepoName) \(repo)"
```

Comment:
This command is executed even with `--log-only true`. I assume it is fine but let me know if you think otherwise.